content/en/observability_pipelines/quickstart/_index.md

```diff
@@ -0,0 +1,184 @@
+Use the load balancers provided by your cloud provider.
+They adjust based on autoscaling events that the default Helm setup is configured for. The load balancers are internal-facing,
+so they are only accessible inside your network.
+
+Use the load balancer URL given to you by Helm when you configure the Datadog Agent.
+
+#### Cross-availability-zone load balancing
+The provided Helm configuration tries to simplify load balancing, but you must take into consideration the potential price implications of cross-AZ traffic. Wherever possible, the samples try to avoid creating situations where multiple cross-AZ hops can happen, preferring performance, simplicity, and cost optimizations at the potential expense of fault tolerance.
+
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+NLBs provisioned by the [AWS Load Balancer Controller][1] are used. It provides
+more flexibility and accuracy than the in-tree controller that AWS provides.
+
+The sample configurations do not enable the "cross-zone load balancing" feature available in this controller. To enable it, add the following annotation to the `service` block:
+
+```
+service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true
+```
+
+[See the AWS LB Controller page for more details.][2]
+
+[1]: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/
+[2]: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/service/annotations/#load-balancer-attributes
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+No special requirements are needed for Azure AKS.
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+Global Access is enabled by default since that is likely required for use in a shared tools cluster.
+{{% /tab %}}
+{{< /tabs >}}
+
+### Buffering
+Observability Pipelines includes multiple buffering strategies that allow you to increase the resilience of your cluster to downstream faults. The provided sample configurations use disk buffers, the capacities of which are rated for approximately 10 minutes of data at 10Mbps/core for Observability Pipelines deployments. That is often enough time for transient issues to resolve themselves, or for incident responders to decide what needs to be done with the observability data.
+
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+For AWS, Datadog recommends using the `io2` EBS drive family. Alternatively, the `gp3` drives can be used as well.
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+For Azure AKS, Datadog recommends using the `default` (also known as `managed-csi`) disks.
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+For Google GKE, Datadog recommends using the `premium-rwo` drive class, because that is backed by SSDs. The HDD-backed class, `standard-rwo`, likely does not provide enough write performance for the buffers to be useful.
+{{% /tab %}}
+{{< /tabs >}}
+
+## Connect the Agent and the Worker
+To send logs and metrics to the Observability Pipelines Worker, update your agent configuration with the following:
+
+```yaml
+vector:
```

Comment:
Noting, this needs to change to `observability_pipelines_worker`. Pierre Prognant was the engineer that implemented this and he is no longer with us. I wish it was as simple as a rename, but I don't think it is based on [this commit](https://github.com/DataDog/datadog-agent/commit/48752386c6ab041b70f1ca8c7ffeec5586595edc).